config_examples/k8scombined-gateway.yaml

```diff
@@ -0,0 +1,125 @@
+extensions:
+  health_check:
+    endpoint: 0.0.0.0:13133
+
+receivers:
```

Comment:
i think it would be better from a data flow persepective to have the gateway collector receive and forward the traffic from the agents. So i would add the otlp receiver here and add a second forwarding pipeline for the data

Reply:
adapted